marlowe-playground-client/grammar.ne

```diff
@@ -0,0 +1,180 @@
+@{%
+const moo = require("moo");
+
+const lexer = moo.compile({
+        WS: /[ \t]+/,
+        number: /0|-?[1-9][0-9]*/,
```

Comment:
It would be good if negative numbers could have parenthesis around (not necessarily something to do at lexer level). But otherwise when copying them from Haskell it can fail. Also, this is not a new problem, it was in the previous parser too, but since we are at it...

Reply:
force users to put it in `()`? Wouldn't that be a bit confusing/annoying?

Reply:
Not force allow (even though in some cases maybe we should, but it is not my point). Because in some cases it is not proper Haskell to not have parenthesis around negative numbers. For example, if you have `Bound -2 3`, this is not proper Haskell, it should be `Bound (-2) 3`.

Reply:
ok, done and deployed.